src/Parsers/ParserDeleteQuery.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <Parsers/IParserBase.h>
+
+namespace DB
+{
+
+/** Query like this:
+  * DELETE FROM [db.]name WHERE ...
```

Comment:
I'd like to have setting `lightweight_delete_mutation = 0/1` to make ordinary `ALTER DELETE` queries lightweight. And enable it by default to check all tests.